src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -345,7 +344,7 @@ public void getUsersByUserId(String userId, String lang,
               Criteria idCrit = new Criteria();
               idCrit.addField(USER_ID_FIELD);
               idCrit.setOperation("=");
-              idCrit.setValue(userId);
+              idCrit.setVal(userId);
```

Comment:
Can you replace the complete method by a PgUtil.getById invokation so that this duplicated code gets removed?